assignments/arrays.js

```diff
@@ -63,32 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[33]["car_year"]} ${inventory[33]["car_make"]} ${inventory[33]["car_model"]}`);
```

Comment:
we need to consider the point where an index starts when retrieving data from an array, remember that an index starts counting from 0, with that information in mind to find the car 33, the actual index number is going to be 32.